### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by eliotexi** [Stellarium/stellarium#2123] [General]
Hey, guys

I have an issue or more of a question. I need to get some contact information of Stellarium - an e-mail at best! I have searched their entire website but couln´t find it so I hoped someone might know it here. Or do you know where I can find it?

That would help me a lot!

Sorry, too, I am new to this and don´t really know how this works yet.

**Comment by alex-w** [comment]
You may ask here or you our mailing list - https://groups.google.com/group/stellarium (stellarium@googlegroups.com)
